.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
These were just some additional settings, they should have been removed, let me check...